.github/workflows/code-coverage-test.yml

```diff
@@ -0,0 +1,54 @@
+name: Code Coverage Test
+
+on:
+  pull_request:             # Remove after testing
+  schedule:
+    - cron: "0 0 * * 6"     # Every Saturday
```

Comment:
If the intention is to run this on a branch other than `master` like we do with Jenkins, I think this restriction:

> - Scheduled workflows will only run on the default branch.

from https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule might be a problem.